packages/preview/touying-unistra-pristine/1.0.0/example/example.typ

```diff
@@ -0,0 +1,143 @@
+#import "@preview/touying:0.5.2": *
+#import "../src/unistra.typ": *
+#import "../src/colors.typ": *
+#import "../src/admonition.typ": *
+#import "../src/settings.typ" as settings
```

Comment:
Consider `#import "../lib.typ": *` Instead of importing things separately.

Reply:
I think importing the theme as it is done in template.typ is the most logical choice as the entry point is src/lib.typ. I have made the necessary changes — thanks!